doc/intl/FR_bzh/README.md

```diff
@@ -0,0 +1,92 @@
+[![MuchWow](https://img.shields.io/badge/Much-Wow-yellow.svg)](https://dogecoin.com)
+
+</div>
+
+Dogecoin zo ur c'hripto-moneiz kumuniezhel awenet gant un anv Shiba Inu. Ar meziant Dogecoin Core a ro an tu da bep hini da c'horvoiñ ur skoulm e rouedadoù ar Blockchain Dogecoin hag a implij an hentenn bouchañ Scrypt evit pro Kempennet eo gant Bitcoin Core ha kripto-moneiz all.
+
+Evit gouzout hiroc'h war ar frejoù dre ziouer implijet war ar rouedad Dogecoin, lennit an erbedadenn war ar [frejoù mar plij].(doc/fee-recommendation.md).
+
+**Lec'hienn web:** [dogecoin.com](https://dogecoin.com)
+
+## Arver 💻
+
+Evit kregiñ gant ho peaj gant Dogecoin Core, sellit ouzh ar [sturlevr staliañ](INSTALL.md) hag ouzh an [tutorell loc'hañ](dock/getting-started.md).
+
+An API JSON-RPC pourchaset gant Dogecoin Core zo emteullet hag a c'hall bezañ furchet gant `dogecoin-cli help`, tra ma c'haller gwelet `dogecoin-cli help <command>` titouroù dre ar munud evit pep ur Gallout a rit ivez sellet ouzh an teul [teulioù Bitcoin Core](https://developer.bitcoin.org/reference/rpc/) - a laka ur protokol damheñvel da dalvezout - evit kaout ur stumm a c'haller lenn
+
+### Ar porzhioù
+
+Dogecoin Core a implij dre ziouer ar porzh `22556` evit ar c'hehentiñ par-ouzh-par a zo ezhomm evit sinkronelaat ar Blockchain ouzh ar ".  Ouzhpenn-se e c'haller digeriñ ur porzh JSONRPC. Dre ziouer e vez implijet ar porzh `22555` evit skoulmoù ar rouedad pennañ. Erbediñ a reomp groñs chom hep diskouez ar porzhioù RPC war ar Genrouedad publik. 
+
+| Function | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+## Diorren dibaouez - Moon plan 🌒
+
+Dogecoin Core zo ur meziant frank ha kumuniezhel. Digor eo an argerzh diorren hag a c'haller gwelet evit an holl ; an holl a c'hall gwelet, kendivizout ha labourat war ar meziant.
+ 
+Danvezioù diorren pennañ :
+
+* [GitHub Projects](https://github.com/dogecoin/dogecoin/projects)  a vez implijet evit
+heuliañ al labourioù rakwelet ha war ar stern evit ar stummoù da zont.
+
+* [GitHub Discussion](https://github.com/dogecoin/dogecoin/discussions) a vez implijet evit
+kaozeal eus perzhioù, steuñvet ha n'int ket steuñvet, liammet war un dro ouzh diorroadur ar meziant Dogecoin Core, ar protokolioù dindan hag an oberiant DOGE.
+  
+* [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
+
+### Strategiezh ar stumm
+An niverennoù stumm zo da-heul ar steredeg `` major.minor.patch``.
+ 
+### Skourroù
+Tri seurt skourroù zo er raktres-mañ :
+
+- **master:** Stabil, zo enni ar stumm diwezhañ eus ar *major.minor* handelv.
+- **maintenance:** Stabil, zo enni ar stumm nevesañ eus ar stummoù kent hag a vez dalc'het en oberiant bepred. Furmad : ```<version>-maint``
+- **development:** Stabil, zo enni kod nevez evit ar stummoù rakwelet. Furmad : ```<version>-dev```
+
```

Comment:
Missing the `archive` branch add a translation of `- **archive:** Stable, immutable branches for old versions that no longer change because they are no longer maintained.`